### PR TITLE
docs: fix grammer and polish the docs

### PR DESCRIPTION
**Pull request by liuming-dev**
Signed-off-by: Liu Ming <hit_oak_tree@126.com>

### CODE REVIEW

**Comment by hongchaodeng** [comment]
I have reviewed this PR and the current grammar has no problem.

File: 1.purpose_and_goals.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
 
-The central problem this specification seeks to address is how distributed applications can be composed and handed off to those responsible for operating them. The problem is not so much about how to write programs, but how to take components of a service-oriented (or microservice-oriented) architecture, and streamline the workflow that surrounds such applications.
+The central problem this specification seeks to address is how distributed applications can be composed and handed off to those responsibilities for operating them. The problem is not so much about how to write programs, but how to take components of a service-oriented (or microservice-oriented) architecture, and streamline the workflow that surrounds such applications.
```
Comment (ryanzhang-oss) [inline 1.purpose_and_goals.md:5]: we need a `verb` like responsible here, not a `noun`

File: 1.purpose_and_goals.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-For example, a contemporary cloud application may be composed of dozens of microservices, each responsible for a discrete chunk of what, broadly speaking, is "an application." Such applications need to be configured, deployed, audited, updated, and deleted. Sometimes the application must be treated as a whole, and sometimes finer levels of granularity are required. And most importantly, often such applications are managed not by one person or one team, but by multiple teams who must cooperate to achieve reliability, stability, and timeliness.
+For example, a contemporary cloud application may be composed of dozens of microservices, each responsibility for a discrete chunk of what, broadly speaking, is "an application." Such applications need to be configured, deployed, audited, updated, and deleted. Sometimes the application must be treated as a whole, and sometimes finer levels of granularity are required. And most importantly, often such applications are managed not by one person or one team, but by multiple teams who must cooperate to achieve reliability, stability, and timeliness.
```
Comment (ryanzhang-oss) [inline 1.purpose_and_goals.md:7]: what exactly has changed?

File: 2.overview_and_terminology.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ This specification proposes a model that defines cloud native applications as fo
 
 > A cloud native application is a collection of interrelated, but discrete _components_ (services, tasks, workers) that, when coupled with configuration and instantiated in a suitable runtime, together accomplish a unified functional purpose.
 
-This specification defines the following application constructs for application developers and application operators to design and operate applications:
+This specification defines the following application constructed for application developers and application operators to design and operate applications:
```
Comment (ryanzhang-oss) [inline 2.overview_and_terminology.md:23]: why use passive voice?

File: introduction.md [hunk 4]
```diff
@@ -8,17 +8,17 @@ In the following, we go through a story that describes an application delivery l
...
+The story begins with the application developers who create an application, such as an online shopping application. They know how to write and test the code. The application program takes a few parameters such as log level and HTTP port. To let the developers focus on implementing the application's business logic, we have application operators (either human or automated operation platforms) take care of operational tasks. This provides a "serverless" experience to application developers: they only need to develop and package the application, and then deliver it to application operators to operate.
```
Comment (ryanzhang-oss) [inline introduction.md:13]: what is changed here?

File: introduction.md [hunk 5]
```diff
@@ -39,8 +39,8 @@ The following diagram demonstrates the platform architecture:
 
 ## For platform builders
 
-Focused on the separation of development concerns from operational considerations through a platform-agnostic specification, the _Open Application Model_ brings modular, extensible, and portable design to building and delivering applications on platforms like Kubernetes.
+Focused on the separation of development concerns from operational considerations through a platform-agnostic specification, the _Open Application Model_ brings modular, extensible, and portable design to build and deliver applications on platforms like Kubernetes.
```
Comment (ryanzhang-oss) [inline introduction.md:42]: you cannot use verbs to form the object of a sentence

Comment (liuming-dev) [reply introduction.md:42]: I don't think so, possibly we need an english expert to figure out this.

File: introduction.md (hunk 4 above)
Comment (liuming-dev) [reply introduction.md:13]: A stop character at the end of the sentence.

File: 2.overview_and_terminology.md (hunk 3 above)
Comment (liuming-dev) [reply 2.overview_and_terminology.md:23]: I think the meaning is that 

*This specification defines the following application **(that is)** constructed for application developers and application operators to design and operate applications:*

File: 1.purpose_and_goals.md (hunk 2 above)
Comment (liuming-dev) [reply 1.purpose_and_goals.md:7]: Just a space character.

File: 1.purpose_and_goals.md (hunk 1 above)
Comment (liuming-dev) [reply 1.purpose_and_goals.md:5]: Yes, I see , it means that 

*The central problem this specification seeks to address is how distributed applications can be composed and handed off to those **(that are)** responsible for operating them.*, this should be `responsible`.

File: 2.overview_and_terminology.md (hunk 3 above)
Comment (ryanzhang-oss) [reply 2.overview_and_terminology.md:23]: constructs here is a `noun`

File: introduction.md (hunk 5 above)
Comment (ryanzhang-oss) [reply introduction.md:42]: This is reviewed by a MSFT technical writer. I don't know who can be called an "English expert" if she is not.....

Comment (liuming-dev) [reply introduction.md:42]: Sorry for bother, please forgive my poor english.

**Pull request closed without merging**